hardhat-tutorial/contracts/Exchange.sol

```diff
@@ -97,7 +97,7 @@ contract Exchange is ERC20 {
         // Burn the sent `LP` tokens from the user's wallet because they are already sent to 
         // remove liquidity
         _burn(msg.sender, _amount);
-        // Transfer `ethAmount` of Eth from user's wallet to the contract
+        // Transfer `ethAmount` of Eth from the contract to the user
         payable(msg.sender).transfer(ethAmount);
         // Transfer `cryptoDevTokenAmount` of `Crypto Dev` tokens from the user's wallet to the contract 
```

Comment:
Forgot to correct this line also. The contract that is transferring the Tokens to the user, not the user sending tokens to the contract

Reply:
great, could you fix this